apps/src/StudioApp.js

```diff
@@ -3423,8 +3425,10 @@ StudioApp.prototype.isNotStartedLevel = function(config) {
     return config.readonlyWorkspace && !config.channel;
   } else {
     return (
-      config.readonlyWorkspace &&
-      progress.levelProgress[progress.currentLevelId] === undefined
+      (config.readonlyWorkspace &&
+        progress.progressByLevel[progress.currentLevelId] === undefined) ||
+      progress.progressByLevel[progress.currentLevelId].status ===
```

Comment:
Noticing some test failures because progress.progressByLevel[progress.currentLevelId] is undefined (also because `progress` is undefined, but that's a testing issue since `progress` was shortcut by `config.readonlyWorkspace` and we never needed to mock it in the past). 
Given that `progress.progressByLevel[progress.currentLevelId]` is expected to sometimes be undefined in 3429, should we also be checking if it's undefined in 3430?

Reply:
I realize that comment got a bit self-referential. The short story: I think 3430 should be  `progress.progressByLevel[progress.currentLevelId] &&  progress.progressByLevel[progress.currentLevelId].status` or something similar.

Reply:
✔️